overthrower/overthrower.cpp

```diff
@@ -208,6 +210,12 @@ extern "C" int deactivateOverthrower()
 #if defined(__APPLE__)
     return 0;
 #else
+    if (!allocated.empty()) {
+        fprintf(stderr, "leaked blocks:.\n");
```

Comment:
I would write: "overthrower has detected not free memory blocks with following addresses:\n".